apps/mail/lib/email-utils.client.tsx

```
@@ -1,59 +1,59 @@
-					case 'post':
-						const controller = new AbortController();
-						const timeoutId = setTimeout(
-							() => controller.abort(),
-							10000, // 10 seconds
-						);
+        switch (listUnsubscribeAction.type) {
+          case 'get':
+            window.open(listUnsubscribeAction.url, '_blank');
+            break;
+          case 'post':
+            const controller = new AbortController();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)